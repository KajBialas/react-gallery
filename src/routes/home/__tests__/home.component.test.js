import React from 'react';
import { shallow } from 'enzyme';
import HomeComponent from '../home.component';

describe('HomeComponent', () => {
  it('should shallows correctly', () => {
    const component = shallow(<HomeComponent />);

    expect(component).toMatchSnapshot();
  });
});