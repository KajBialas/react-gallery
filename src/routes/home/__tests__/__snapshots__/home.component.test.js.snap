// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should shallows correctly 1`] = `
<Fragment>
  <HeaderComponent />
  <styled.div>
    <styled.div>
      This is react example gallery.
    </styled.div>
  </styled.div>
</Fragment>
`;
